src/item.h

```diff
@@ -324,6 +324,7 @@ class item : public visitable<item>
         bool ready_to_revive( const tripoint &pos ) const;
 
         bool is_money() const;
+        bool is_software() const;
```

Comment:
```suggestion
        bool is_software() const;
        bool is_software_storage() const;
```
Add `is_software_storage`.